release-notes/0.4.10.rst

```diff
@@ -0,0 +1,7 @@
+0.4.10 (2024-09-17)
+==================
```

Comment:
RST is obnoxious. The underline under the heading must be at least as long as the text above it

```suggestion
0.4.10 (2024-09-17)
===================
```

Reply:
Thanks a lot @Eric-Arellano !